.github/workflows/sbom.yaml

```diff
@@ -0,0 +1,42 @@
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        type: string
+        required: true
+
+permissions: {}
+
+jobs:
+  test_bom_action:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write
+    name: Install bom and generate SBOM
+    steps:
+      - name: Set tag name
+        shell: bash
+        run: |
+          TAG=${{ github.event.inputs.tag }}
+          echo "TAG=$TAG" >> "$GITHUB_ENV"
+      - name: Install bom
+        uses: kubernetes-sigs/release-actions/setup-bom@main
+      - name: Checkout repository
+        uses: actions/checkout@v4
+      - name: Generate SBOM
+        run: |
+          bom generate -o /tmp/kueue.spdx .
+          cd /tmp
+          tar zcf sbom.tar.gz *.spdx
+      - name: Generate SBOM hash
+        shell: bash
+        run: |
+          # sha256sum generates sha256 hash for sbom.
+          # base64 -w0 encodes to base64 and outputs on a single line.
+          # sha256sum /tmp/sbom.tar.gz ... | base64 -w0
+          echo "hashes=$(sha256sum /tmp/sbom.tar.gz | base64 -w0)" >> "$GITHUB_OUTPUT"
```

Comment:
This is just a question. Where do we use this `hashes` environment variable in this workflow?
As I can see, there aren't places to use this `hashes` environment variable.

Reply:
PING @psschwei